websocket/connection_test.go

```diff
@@ -379,7 +379,7 @@ func TestDurableConnectionWhenConnectionBreaksDown(t *testing.T) {
 
 func TestDurableConnectionSendsPingsRegularly(t *testing.T) {
 	// Reset pongTimeout to something quite short.
-	const pongTimeout = 100 * time.Millisecond
+	pongTimeout = 100 * time.Millisecond
```

Comment:
do we want to restore this afterwards? (I guess it doesn't matter right now since no other tests are using this, but theoretically that could change 🤷)

Reply:
Done.